src/nl/rubensten/texifyidea/settings/LabelingCommandInformation.kt

```diff
@@ -0,0 +1,23 @@
+package nl.rubensten.texifyidea.settings
+
+data class LabelingCommandInformation(var commandName: String, var position: Int, var labelPrevCmd: Boolean) {
+    companion object {
+        /**
+         * Needed to get bck the object from the string in the xml file
+         */
+        @JvmStatic fun fromString(string: CharSequence): LabelingCommandInformation {
+            val parts = string.split(";")
+            if (parts.size != 3) {
+                throw IllegalArgumentException("String must contain exactly three parts")
+            }
+            val position = parts[1].toIntOrNull() ?: throw IllegalArgumentException("Second part must be type Int")
```

Comment:
Not wrong in the slightest, but more Kotlin idiom:
```suggestion
            val position = parts[1].toIntOrNull() ?: error("Second part must be type Int")
```